assets/js/src/marketing_optin_block/frontend.tsx

```diff
@@ -0,0 +1,17 @@
+/**
+ * External dependencies
+ */
+/* eslint-disable react/react-in-jsx-scope */
```

Comment:
I don't think this file uses react, does it? Is this redundant? And the file itself can have a `.ts` extension.

Reply:
This is another case where your repository linting rules throw errors if react is not imported. We don't need react import here.

Reply:
I believe that in this particular case renaming the file to `frontend.ts` might be enough to get rid of the linter warning.